ui/app/AppLayouts/Onboarding/popups/UploadProfilePicModal.qml

```diff
@@ -28,16 +29,23 @@ ModalPopup {
     Item {
         anchors.fill: parent
 
-        RoundedImage {
+        StatusRoundedImage {
```

Comment:
You could even use StatusSmartIdenticon here.

Reply:
@endulab yes, but the thing is that under the hood `StatusSmartIdenticon` is again `StatusRoundedImage` and if I used `StatusSmartIdenticon` I wouldn't be able to set transparent background and border for the image at the same time, cause they depends on `isIdenticon` prop, and if I set it to true then I would get black background , if I set it to false then I would not have border.